access/slack/bot.go

```diff
@@ -41,6 +41,10 @@ func NewBot(conf Config, clusterName, webProxyAddr string) (Bot, error) {
 		if webProxyURL, err = url.Parse(webProxyAddr); err != nil {
 			return Bot{}, err
 		}
+		if webProxyURL.Scheme == "https" && webProxyURL.Port() == "443" {
+			// Cut off redundant :443
+			webProxyURL.Host = webProxyURL.Hostname()
```

Comment:
Why check for a specific port, wouldn't .Hostname() always work? Even if there is no port, it should return itself?

Reply:
Can you also please cover by tests

Reply:
> Even if there is no port, it should return itself?

Yes but if there's a port *and* it's not 443, I need to preserve it.

Reply:
Why preserve it?

Reply:
It's not strictly necessary but links like `https://evilmartians.teleport.sh:443` look weird since `https` already means `443` by default. I wanted to slightly improve the UI by cutting off redundant `:443` part in HTTPS links. However if the port differs from 443 it must be preserved in the URL.